### PR TITLE
Update confirmation page to better match design

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

See title.

Use the `CartInventory` component to display purchase summary.

Refactor the reusable component a bit to be more flexible, use render props.

## Justify why they are needed

Would be nice to have a single way to display the inventory of a cart.

This is far from done but brings us closer to where we want to be.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #888** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/888" target="_blank">Graphite</a> 
    * **PR #890** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/890" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/890?utm_source=stack-comment).

File: apps/store/src/components/ConfirmationPage/ConfirmationPage.helpers.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ const TODAY = new Date()
 export const fromNow = (dateObj: Date, locale: string) => {
   const diff = Math.abs(TODAY.getTime() - dateObj.getTime())
   const diffDays = Math.floor(diff / (1000 * 3600 * 24))
-  return diffDays === 0 ? 'Today' : dateObj.toLocaleDateString(locale)
+  return diffDays === 0 ? 'today' : dateObj.toLocaleDateString(locale)
```
Comment (alebedev) [inline apps/store/src/components/ConfirmationPage/ConfirmationPage.helpers.ts:6]: Let's localize "today" too

**Review by alebedev — APPROVED** [review]
Looks good

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/890?utm_source=stack-comment).


Job Reference: KrNNugJV01lmDDBZCLzA

**Pull request merged**